requirements-dev.txt

```diff
@@ -0,0 +1,2 @@
+black
+pre-commit
```

Comment:
So if a new member cannot commit things, does the system tell him/her that it's because things in this files are not installed? Since we have only two, what if we just put them in requirements.txt?